### PR TITLE
Typo fix (#6)

### PR DESCRIPTION
**Pull request by Laurits7**
As @andreypz pointed out in the comment of PR cms-nanoAOD#251, then there was a mistake in one of the print statements that was introduced in that PR. Added the str() to the print statement and made some further pep8.

### CODE REVIEW

**Comment by mariadalfonso** [comment]
@andreypz @camclean 
Please check if this PR fixes the two issues you reported.
notify the outcome here.

**Comment by camclean** [comment]
Looks good. Thanks!

**Comment by andreypz** [comment]
I did not have time to check this, but looking at the code, it should be fixed.
In general, someone should just check that [crab submission](https://github.com/cms-nanoAOD/nanoAOD-tools/tree/master/crab) still works after this PR (in CMSSW using python2).

File: python/postprocessing/framework/postprocessor.py [hunk 1]
```diff
@@ -127,10 +127,10 @@ def run(self):
                     "Running with --noout and no modules does nothing!")
 
         # Open histogram file, if desired
-        if (self.histFileName != None and self.histDirName == None) or (self.histFileName == None and self.histDirName != None):
+        if (self.histFileName is not None and self.histDirName is None) or (self.histFileName is None and self.histDirName is not None):
             raise RuntimeError(
                 "Must specify both histogram file and histogram directory!")
-        elif self.histFileName != None and self.histDirName != None:
+        elif self.histFileName is not None and self.histDirName is None:
```
Comment (nurfikri89) [inline python/postprocessing/framework/postprocessor.py:133]: Hi @Laurits7,  shouldn't this be  
```python
self.histDirName is not None
```
at the end?

Comment (nurfikri89) [reply python/postprocessing/framework/postprocessor.py:133]: Fixed in PR #263

**Pull request merged**